### PR TITLE
fix(argo-cd): AWS GRPC Service Type

### PR DESCRIPTION
**Pull request by toneill818**
The Service Type needed depends on the annotation `alb.ingress.kubernetes.io/target-type` on the ingress. This PR allows users to set the service type to Cluster IP if they are using IP Mode for the ingress. By default the target type is instance and requires the service to be of type NodePort.

https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.2/how-it-works/#ingress-traffic

Checklist:

* [X] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#versioning)
* [X] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#changelog).
* [X] Any new values are backwards compatible and/or have sensible default.
* [X] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo).
* [ ] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

**Comment by mbevc1** [comment]
Commits need sign-off as well for test to pass.

**Comment by toneill818** [comment]
@mbevc1 The [commit](https://github.com/argoproj/argo-helm/pull/862/commits/98192beacd13608ff0516be12619db3030d04974) that is failing has a sign-off, but the DCO bot does not like the email address. Is this something I can fix or is it an error with the DCO bot?

**Comment by mbevc1** [comment]
Huh, that interesting, seems like it doesn't like applied comments from suggestions :slightly_smiling_face: Wonder if rebasing might help?

Also version needs bumping now as well due PR merges.

**Comment by toneill818** [comment]
@mbevc1 Tried a rebase and it looks like it messed some stuff up. I will create a new branch off of master and create a new PR for this fix, hopefully that will fix the DCO issue.

**Comment by toneill818** [comment]
@mbevc1 All checks have now passed.

**Review by mbevc1 — APPROVED** [review]

**Comment by mbevc1** [comment]
Thanks! :+1:

File: charts/argo-cd/Chart.yaml
Comment (mkilchhofer) [inline charts/argo-cd/Chart.yaml:5]: Unfortunately, the changelog annoation was not updated... 
Friendly reminder: Please keep changelog annoation update in mind before merge @mbevc1 :-)

Comment (mbevc1) [reply charts/argo-cd/Chart.yaml:5]: Ah, sorry totally missed that after last rebase as there were issues with signing commits.

**Pull request merged**
